user-interface/models/survey-modal.js

```diff
@@ -0,0 +1,91 @@
+const { getSurveyQuestions } = require('../../services/strapi/index');
+
+async function getSurveyModal(username) {
+  const SurveyQuestions = await getSurveyQuestions();
+  const questions = SurveyQuestions.map((SurveyQuestion) => ({
+    type: 'input',
+    element: {
+      type: 'static_select',
+      placeholder: {
+        type: 'plain_text',
+        text: 'I feel..',
+        emoji: true,
+      },
+      options: [
```

Comment:
Make the options dynamic as well in surveys as well as daily checkin

Reply:
done